app/views/devise/shared/_links.html.erb

```diff
@@ -6,10 +6,6 @@
   <%= link_to "Sign up", new_registration_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
```

Comment:
i think you should just comment it out instead of deleting in case we need it in the future (like if this proj doesn't die?)